app/controllers/v0/facilities/ccp_controller.rb

```diff
@@ -36,6 +29,37 @@ def services
 
   private
 
+  def ppms_search
+    if Flipper.enabled?(:facility_locator_ppms_legacy_urgent_care_to_pos_locator)
```

Comment:
does there need to be a test for when this flipper is disabled that shows the `261QU0200X` result is the same as `cc_provider`?

Reply:
I removed the test that showed how it worked without flipper. if you need it I can put it back.

Reply:
As long as it's a valid code path I think it's worth keeping